engines/ags/dialogs.cpp

```diff
@@ -75,6 +76,9 @@ AGSOptionsWidget::AGSOptionsWidget(GuiObject *boss, const Common::String &name,
 		_langPopUp->appendEntry(traFileName, i++);
 	}
 
+	// Override game save management
+	_overrideSavesCheckbox = new GUI::CheckboxWidget(widgetsBoss(), _dialogLayout + ".savesOvr", _("Force ScummVM save management"), _("Never disable ScummVM save management and autosaves.\nNOTE: This could cause save duplication and other oddities"));
+
```

Comment:
```suggestion
	// Override game save management
	if (Common::checkGameGUIOption(GAMEOPTION_NO_AUTOSAVE, ConfMan.get("guioptions", domain)) ||
		Common::checkGameGUIOption(GAMEOPTION_NO_SAVELOAD, ConfMan.get("guioptions"))) {
		_overrideSavesCheckbox = new GUI::CheckboxWidget(widgetsBoss(), _dialogLayout + ".savesOvr", _("Force ScummVM save management"), _("Never disable ScummVM save management and autosaves.\nNOTE: This could cause save duplication and other oddities"));
	else
		_overrideSavesCheckbox  = nullptr;

```
And add a sanity check where `_overrideSavesCheckbox` is used to avoid dereferencing a null pointer.

I assume the checkbox only make sense for games that have those flags?